src/utils/components/DiskModal/components/DiskSourceSelect/components/DiskSourceSelectOptions/DiskSourceSelectOptionGroups.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { FC } from 'react';
+
+import { isEmpty } from '@kubevirt-utils/utils/utils';
+import { Divider, SelectGroup } from '@patternfly/react-core';
+
+import { DiskSourceOptionGroup } from '../../utils/types';
+
+import DiskSourceSelectOptions from './DiskSourceSelectOptions';
+
+type DiskSourceSelectOptionGroupsProps = {
+  isTemplate?: boolean;
+  isVMRunning: boolean;
+  optionGroups: DiskSourceOptionGroup[];
+};
+
+const DiskSourceSelectOptionGroups: FC<DiskSourceSelectOptionGroupsProps> = ({
+  isTemplate,
+  isVMRunning,
+  optionGroups,
+}) => {
+  return (
+    <>
+      {optionGroups.map((option, index) => {
+        const { groupLabel, items } = option;
+        const isLast = index === optionGroups.length - 1;
+
+        const children = (
+          <DiskSourceSelectOptions
+            isTemplate={isTemplate}
+            isVMRunning={isVMRunning}
+            items={items}
+          />
+        );
+
+        if (!isEmpty(groupLabel)) {
```

Comment:
instead of a negative check u can check
isEmpty
return <>
            {children}
            {!isLast && <Divider />}
          </>